src/components/validations/validation-icon.spec.js

```diff
@@ -10,8 +10,8 @@ import Icon from '../icon';
 
 describe('ValidationIcon', () => {
   it('renders with an icon for the given type', () => {
-    const wrapper = mount(<ValidationIcon type='error' />);
-    expect(wrapper.find(ValidationIconStyle).prop('validationType')).toEqual('error');
+    const wrapper = mount(<ValidationIcon type='error' />).find('div');
+    expect(wrapper.prop('data-validation-type')).toEqual('error');
```

Comment:
`data-validation-type` was only used for testing assertions. It was decided that this shouldn't be in the DOM so it was removed. Because of this, your suggestion would not work.